apps/web/cypress/support/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { HttpResponseInterceptor, RouteMatcher, StaticResponse } from 'cypress/types/net-stubbing';
+
+export function interceptIndefinitely(
+  requestMatcher: RouteMatcher,
+  response?: StaticResponse | HttpResponseInterceptor
+): { sendResponse: () => void } {
+  let sendResponse;
+  const trigger = new Promise((resolve) => {
+    sendResponse = resolve;
+  });
+
+  cy.intercept(requestMatcher, (request) => {
+    return trigger.then(() => {
+      request.reply(response);
+    });
+  });
+  return { sendResponse };
+}
```

Comment:
Is this one is in use? If not I think better we remove it

Reply:
ok